packages/orbit-components/src/List/ListContext.ts

```diff
@@ -1,18 +0,0 @@
-import * as React from "react";
-
-import type { Size } from "../common/types";
-import type { Type } from "./types";
-
-interface ListContextProps {
-  readonly size?: Size | null;
-  readonly type?: Type | null;
-}
-
-const ListContext = React.createContext<ListContextProps>({
```

Comment:
Not sure if it's a good idea to remove the context here. It's being used in order not to pass those size and type properties to each ListItem individually, so better for users, as they can have a lot of them. 

And it's also a breaking change :) So I think better will be to keep it